ui/src/views/PageNotFoundView.vue

```diff
@@ -41,6 +55,16 @@
     mainPicture() {
       return require("@assets/svg/404.png");
     }
+  },
+  methods: {
```

Comment:
One way is to do this, the other way is to use CSS and target the media query instead. But this is fine, I will make a global mixin so that we dont have to specify this in every file.